types/react-relay/test/relay-hooks-tests.tsx

```diff
@@ -593,177 +592,179 @@ function PaginationFragment_WithNonNullUserProp() {
-                    <div>
-                        {node.name} - {node.age}
-                    </div>
-                ))}
-
-                <button onClick={() => loadNext(10)}>Load more friends</button>
-            </>
-        );
-    };
-}
-
-function BlockingPaginationFragment_WithNonNullUserProp() {
-    interface FriendsListPaginationQueryVariables {
-        count?: number;
-        cursor?: string;
-        id: string;
-    }
-
-    interface FriendsListPaginationQueryResponse {
-        readonly node: {
-            readonly ' $fragmentRefs': FragmentRefs<'FriendsListComponent_user'>;
-        };
-    }
-
-    interface FriendsListPaginationQuery {
-        readonly response: FriendsListPaginationQueryResponse;
-        readonly variables: FriendsListPaginationQueryVariables;
-    }
-
-    interface FriendsListComponent_user {
-        readonly name: string;
-        readonly friends: {
-            readonly edges: ReadonlyArray<{
-                readonly node: {
-                    readonly name: string;
-                    readonly age: number;
-                };
-            }>;
-        };
-        readonly id: string;
-        readonly ' $refType': 'FriendsListComponent_user';
-    }
-
-    type FriendsListComponent_user$data = FriendsListComponent_user;
-
-    interface FriendsListComponent_user$key {
-        readonly ' $data'?: FriendsListComponent_user$data;
-        readonly ' $fragmentRefs': FragmentRefs<'FriendsListComponent_user'>;
-    }
-
-    interface Props {
-        user: FriendsListComponent_user$key;
-    }
-
-    return function FriendsList(props: Props) {
-        const {
-            data,
-            loadNext,
-            loadPrevious,
-            hasNext,
-            hasPrevious,
-            refetch, // For refetching connection
-        } = useBlockingPaginationFragment<FriendsListPaginationQuery, FriendsListComponent_user$key>(
-            graphql`
-                fragment FriendsListComponent_user on User @refetchable(queryName: "FriendsListPaginationQuery") {
-                    name
-                    friends(first: $count, after: $cursor) @connection(key: "FriendsList_user_friends") {
-                        edges {
-                            node {
-                                name
-                                age
-                            }
-                        }
-                    }
-                }
-            `,
-            props.user,
-        );
-
-        return (
-            <>
-                <h1>Friends of {data.name}:</h1>
-
-                {data.friends.edges.map(({ node }) => (
-                    <div>
-                        {node.name} - {node.age}
-                    </div>
-                ))}
-
-                <button onClick={() => loadNext(10)}>Load more friends</button>
-            </>
-        );
-    };
-}
+// useBlockingPaginationFragment currently commented out, this api isnt currently exported by react-relay
```

Comment:
@rbalicki2 correct that these are not exported? Couldn't find them in the hooks or anything.

Reply:
Correct. These aren't exported. They will be, when React with concurrent mode is available.

Might make sense to delete the types, and get them back from the history, or just leave them commented. /shrug